install.sh

```
@@ -0,0 +1,247 @@
+eslint_setup ()
+{
+    config_file_extension="$1"
+    config_contents="$2"
+
+    expected_config_file="eslint.config.${config_file_extension}"
+    expected_config_match="from '@verkstedt/lint/eslint'"
+    existing_config_files=$(
+        find . -maxdepth 1 -type f \( -name '.eslintrc' -or -name '.eslintrc.*' -or -name 'eslint.config.*' \)
+    )
+
+    if [ -n "$( jq '.eslintConfig // empty' package.json > /dev/null 2>&1 )" ]
```

Shellcheck (0.11.0)

[warning] This command substitution will be empty because the command's output gets redirected away.

(SC2327)